rg_instructor_analytics/templates/rg_instructor_analytics/instructor_analytics_fragment.html

```diff
@@ -44,15 +51,15 @@
                 </ul>
 
                 <section id="section-enrollment-stats" class="idash-section">
-                    <%include file="enrollment_stats.html" args="section_data={}" />
+                        <%include file="enrollment_stats.html" args="section_data=enroll_info" />
```

Comment:
too many spaces please use 4 spaces indentation here and below.

Reply:
It is autoformatting for the blocks of python code.

Reply:
autoformatting is not an answer please make indentation equal to 4 spaces.

Reply:
done